pkg/api/handlers/libpod/images.go

```diff
@@ -416,3 +419,84 @@ func ImagesPull(w http.ResponseWriter, r *http.Request) {
 
 	utils.WriteResponse(w, http.StatusOK, res)
 }
+
+func CommitContainer(w http.ResponseWriter, r *http.Request) {
+	var (
+		destImage string
+		mimeType  string
+	)
+	decoder := r.Context().Value("decoder").(*schema.Decoder)
+	runtime := r.Context().Value("runtime").(*libpod.Runtime)
+
+	query := struct {
+		Author    string `schema:"author"`
+		Changes   string `schema:"changes"`
+		Comment   string `schema:"comment"`
+		Container string `schema:"container"`
+		Format    string `schema:"format"`
+		Pause     bool   `schema:"pause"`
+		Repo      string `schema:"repo"`
+		Tag       string `schema:"tag"`
+	}{
+		Format: "oci",
+	}
+
+	if err := decoder.Decode(&query, r.URL.Query()); err != nil {
+		utils.Error(w, "Something went wrong.", http.StatusBadRequest, errors.Wrapf(err, "Failed to parse parameters for %s", r.URL.String()))
+		return
+	}
+	rtc, err := runtime.GetConfig()
+	if err != nil {
+		utils.Error(w, "Something went wrong.", http.StatusInternalServerError, errors.Wrap(err, "Decode()"))
```

Comment:
I still don't like the "Something went wrong" line.

Reply:
We also have `utils.InternalServerError()` that abstracts this away...